lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/DuplicatableMessage.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.objects;
+
+import org.eclipse.mosaic.lib.objects.v2x.MessageRouting;
+
+/**
+ * Creates a copy of the message
+ * @param <MessageT>
```

Comment:
As the type is declared as `T` this should say `@param <T>`.
Please also add a short description for the type

Reply:
done